RELEASE.md

```diff
@@ -0,0 +1,63 @@
+# Open Food Facts - Flutter App - Release Guide
+
+Welcome to the release guide for the Open Food Facts Flutter App. Below are the steps to guide you through our 2 weeks release process.
+
+## 1. Pre-release Checklist
+
+Before you proceed with the release, ensure the following:
+
+- [ ] All the features and fixes planned for this release are merged into the `main` branch.
+- [ ] All merged features and fixes have corresponding unit and integration tests.
+- [ ] Update version number in the `pubspec.yaml` according to [semantic versioning](https://semver.org/). (Mergin Release Please should take care of that, and also provoke a F-Droid release up to a week later.)
```

Comment:
This is one is false, as it's done by the GitHub Action when we release.
I've already tried to say it was a bad idea, but the answer was a NO